hack/jenkins/test-flake-chart/sync_tests.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
```

Comment:
add a comment block what does this script do ?

Reply:
Done.